### PR TITLE
add retention commands

### PR DESCRIPTION
**Pull request by fblackburn1**

### CODE REVIEW

File: wazo_call_logd_client/commands/retention.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            headers['Wazo-Tenant'] = tenant_uuid
+
+        url = self._client.url('retention')
+        r = self.session.get(url, headers=headers)
+        self.raise_from_response(r)
+        return r.json()
+
+    def update(self, tenant_uuid=None, **body):
+        tenant_uuid = tenant_uuid or self._client.tenant()
+        headers = self.headers.copy()
```
Comment (pc-m) [inline wazo_call_logd_client/commands/retention.py]: Should the headers contain 'Content-Type: application/json' on the `PUT`?

Comment (fblackburn1) [reply wazo_call_logd_client/commands/retention.py]: I think request will add it automatically, but not sure if it's when headers is None (which is not the case here)
Anyways, fixed

Comment (fblackburn1) [reply wazo_call_logd_client/commands/retention.py]: oups wait it's a bad refactor

Comment (fblackburn1) [reply wazo_call_logd_client/commands/retention.py]: Now really fixed

**Review by pc-m — APPROVED** [review]

**Pull request merged**